pixel-saver@deadalnix.me/buttons.js

```diff
@@ -54,8 +54,13 @@ function createButtons() {
 	
 	orders[0] = orders[0].split(',');
 	
-	// Check if it's actually exists, if not then create it
-	if(typeof orders[1] == 'undefined') orders[1] = '';
+	// Check if it actually exists, if not then create it
+	if(typeof orders[1] == 'undefined') {
+		orders[1] = '';
+	// If the button layout is explicit defined as empty, do nothing
+	} else if(typeof orders[1] != 'undefined' && orders[1].length == 0) {
```

Comment:
The check for `typeof orders[1] != 'undefined'` is redundant here due to the `else` keyword and the previous `if` statement.

Reply:
Ah, yes, will fix that